### PR TITLE
fix upgrade test of 2019.1.4

### PR DESCRIPTION
**Pull request by amoskong**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by amoskong** [comment]
The PR is tested by https://jenkins.scylladb.com/job/scylla-staging/job/amos/job/rolling-upgrade-2019.1/job/rolling-upgrade-centos7/7/

File: jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ rollingUpgradePipeline(
     params: params,
 
     backend: 'gce',
-    base_versions: ['2019.1', '3.1'],
+    base_versions: ['2018.1', '2019.1', '3.0'],
```
Comment (amoskong) [inline jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile]: @roydahan  do we need also add 2.3 here?

However, upgrade from 2.3 to 2019.1.4 isn't supported by ScyllaDB.

**Comment by roydahan** [comment]
Looks good but why is it directly to branch-2019.1.4?

**Comment by amoskong** [comment]
> Looks good but why is it directly to branch-2019.1.4?

The upgrade version change only targets to 2019.1.4.
The workaround fix of 2018.1 is only used by 2019.1.4 right now, so I didn't send it to master.

**Pull request merged**